src/css/components/panel.scss

```diff
@@ -22,6 +22,17 @@ $panel-padding: $grid-3;
   .tableWrapper:last-child {
     margin-bottom: 2rem;
   }
+  .subtext {
```

Comment:
K. So we have to try and stop adding things under `.panel`. Is `subtext` an existing class? What's it's purpose? Rather the adding different styling for the `h2`, it might make more sense to add a `sans-s3` class to the `h2` in the html.